app/views/layouts/_sidebar_tags.html.erb

```diff
@@ -3,7 +3,7 @@
     <header class="p-2 pr-0 flex items-center justify-between">
       <h3 class="crayons-subtitle-3">My Tags</h3>
       <a id="tag-priority-link" href="/dashboard/following_tags" class="crayons-btn crayons-btn--icon crayons-btn--ghost-dimmed" aria-label="Customize tag priority" title="Customize tag priority">
-        <%= inline_svg_tag("cog.svg", class: "crayons-icon") %>
+        <%= inline_svg_tag("twemoji/customization.svg", aria: true, class: "crayons-icon crayons-icon--default") %>
```

Comment:
This is just to be consistent with other cog icons we used elsewhere.

Reply:
Just as a heads up, we still use the other cog icon for the V2 editor options & the comment subscription component (I see it around the site more than this one it's being replaced with 😄 )

Reply:
I've just realized using this twemoji cog doesn't really make sense therefore I reverted this change... Sorry for confusion!